src/AdoNetCore.AseClient/Internal/InternalConnection.cs

```diff
@@ -482,7 +482,7 @@ public void Dispose()
             _socket.Dispose();
         }
 
-        public static IDictionary EmptyStatistics => new Dictionary<string, long>();
+        public static readonly IDictionary EmptyStatistics = new Dictionary<string, long>();
```

Comment:
That too.

But what I was referring to was that the way it was named made it sound like it was an immutable dictionary. Similar to string.Empty. And you planned to use this to reset statistics periodically. If I’ve understood your intention correctly, then maybe this would be more appropriate: https://msdn.microsoft.com/en-us/library/gg712875(v=vs.110).aspx

Reply:
Ah cool, I didn't know about that one.

(Yeah, intent is an empty readonly set for cases where there are no statistics to report.)